src/Commands/HydroponicsDrushCommands.php

```diff
@@ -44,53 +46,152 @@ public function hydroponics(
       \Drupal::state()->set('hydroponics.heartbeat', $currenttime);
     });
     $active_queues = \Drupal::config('strawberryfield.hydroponics_settings')->get('queues');
+
+    // Store timer per queue
     $done = [];
 
+    // Store child timeout timer
+    $timeout = [];
+
     //track when queue are empty for n cycles
     $idle = [];
 
+    // store data for each child process
+    $child = [];
+
+    // track queue items count
+    $items = [];
+
+    // Parameters (ToDO put on config page?)
+    $queue_check_time = 1;
+    $child_timeout_time = 30;
```

Comment:
When to assume the child did not respond, how long we wait. Maybe this can be a number that comes from each processor